dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -237,19 +237,12 @@ public class ConnectorArguments extends DefaultArguments {
   // specified)").withRequiredArg().describedAs("my_dbname").withValuesSeparatedBy(',');
   private final OptionSpec<File> optionOutput =
       parser
-          .accepts("output", "Output file (cannot be used together with --output-dir)")
+          .accepts(
+              "output",
+              "Output file or directory name. If the file name, along with the `.zip` extension, is not provided dumper will attempt to create the zip file with default file name in the directory")
```

Comment:
nit 1: "default file name" => "the default file name"
nit 2: A dot is missing at the end.